.eslintrc.json

```diff
@@ -21,7 +21,8 @@
     "@typescript-eslint/no-unused-vars": "error",
     "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.test.*", "./*"] }],
     "jsx-a11y/label-has-associated-control": ["error", { "assert": "either" }],
-    "react/jsx-filename-extension": ["error", { "extensions": [".tsx"] }]
+    "react/jsx-filename-extension": ["error", { "extensions": [".tsx"] }],
+    "react/jsx-fragments": "off"
```

Comment:
I see this was logging an error:

```
identity-reporting/src/report-filter-controls.tsx
  91:5  error  Fragments are only supported starting from React v16.2. Please disable the `react/jsx-fragments` rule in ESLint settings or upgrade your version of React  react/jsx-fragments
```

Alternatively, we could change the configured `version` below from `16.0` to `16.2`?

I picked `16.0` based on [official defaults for Preact](https://github.com/preactjs/eslint-config-preact/blob/d9c992501fb9e59d49167298e18fbf3145210abb/index.js#L60-L61), though it seems that `16.2` works perfectly fine, at least for now 🤷

Reply:
Oh that makes sense, I thought the error was just wrong since we use preact instead of react 🤦 will update

Reply:
6642360cb79d08ae63a5faf2a91cdd9407149d6d